lib/Data/Validate/Sanctions/Redis.pm

```diff
@@ -0,0 +1,271 @@
+package Data::Validate::Sanctions::Redis;
+
+use strict;
+use warnings;
+
+use parent 'Data::Validate::Sanctions';
+
+use Data::Validate::Sanctions::Fetcher;
+use Scalar::Util    qw(blessed);
+use List::Util      qw(max);
+use JSON::MaybeUTF8 qw(encode_json_utf8 decode_json_utf8);
+use YAML::XS        qw(DumpFile);
+use Syntax::Keyword::Try;
+
+# VERSION
+
+sub new {
+    my ($class, %args) = @_;
+
+    my $self = {};
+
+    $self->{connection} = $args{connection} or die 'Redis connection is missing';
+
+    $self->{sources} = [keys Data::Validate::Sanctions::Fetcher::config(eu_token => 'dummy')->%*];
+
+    $self->{args}      = {%args};
+    $self->{last_time} = 0;
+    my $object = bless $self, ref($class) || $class;
+    $object->_load_data();
+
+    return $object;
+}
+
+sub last_updated {
+    my $self = shift;
+    my $list = shift;
+
+    if ($list) {
+        return $self->{_data}->{$list}->{updated};
+    } else {
+        $self->_load_data();
+        return max(map { $_->{updated} } values %{$self->{_data}});
```

Comment:
out of curiosity is this equivalent to `return $self->{last_time}`

Reply:
I've copy/psated it from the parent class, where these two were different. But since the funciton is redundant (the same as parent class) I'm going to remove it. Thanks for the hint.